inst/www/css/disconnect-dialogue.css

```diff
@@ -0,0 +1,36 @@
+#ss-connect-dialog { display: none !important; }
+#shiny-disconnected-overlay { display: none !important; }
+
+#ss-overlay {
+             background-color: #000000 !important;
+             opacity: 0.6 !important;
+             position: fixed !important;
+             top: 0 !important;
+             left: 0 !important;
+             bottom: 0 !important;
+             right: 0 !important;
+             z-index: 99998 !important;
+             overflow: hidden !important;
+             cursor: not-allowed !important;
+          }
+
+#custom-disconnect-dialog {
+             background: #ffffff !important;
+             color: #000000 !important;
+             width: full !important;
+             transform: translateX(-50%) translateY(-50%) !important;
+             font-size: 16px !important;
```

Comment:
Should we be setting font size here, or should we let it pull through the standard font stylings from the dashboard itself?

Reply:
Fair, probably a good reason for that but we can worry about doing CSS properly another time as there's plenty more of that to sort out and move into here!

I guess at least with the !important 's being removed a user can overwrite this with their own CSS now if needed